rockcraft/templates/simple/rockcraft.yaml.j2

```diff
@@ -1,7 +1,7 @@
 name: {{name}}
 # see {{versioned_url}}/explanation/bases/
 # for more information about bases and using 'bare' bases for chiselled rocks
-base: ubuntu@22.04 # the base environment for this rock
+base: ubuntu@24.04 # the base environment for this rock
```

Comment:
The django and flask templates are still on `22.04`. Is that intentional to not update them to `24.04`?

Reply:
Yes, I need to check with the "owners" of the 12f whether we want to bump the default base for the extensions.
Not all extensions support all bases, so this should happen in a separate task